lighthouse/tests/beacon_node.rs

```diff
@@ -1264,47 +1267,53 @@ fn network_load_flag() {
 // Tests for ENR flags.
 #[test]
 fn enr_udp_port_flag() {
-    let port = unused_udp4_port().expect("Unable to find unused port.");
+    let port = DUMMY_ENR_UDP_PORT;
+    assert!(port != 0);
```

Comment:
you can prevent all this by simply adding a constant using const panic like this
```rust
const _: () = assert!(DUMMY_ENR_UDP_PORT != 0)
```
(hoping I remember the syntax correctly)